### ISSUE TITLE
Running thread of what is functional

### ISSUE THREAD

**Issue by Brdrker** [mwojtkowski/Ovale#14]
Just asking for a list of what is currently functional in the alpha (or any ongoing) release. Actively using it on my pally (and play tested on DK in frost and unholy) w/o issue. I'm a casual player so I can't ascertain if this mod is taking full advantage of a given spec but it's good to have it teaching me basic flow again in 8.x). Leveling a Monk (upper 20's) and the DPS build seems to be working great so far.

**Comment by mwojtkowski** [comment]
Ovale by itself shouldn't cause any errors
If you get lua errors it is probably caused by outdated spell definitions

As for spells I've updated
- Death Knight
- Demon Hunter
- Monk
- Paladin
all above have imported simc profiles

**Comment by Brdrker** [comment]
If someone could tell me how to put together the spell lists (aka where to find spellID, etc.) and field a few questions, I'd be happy to help as well. Currently unemployed so looking for something to do.

**Comment by jj1232727** [comment]
Wowhead

**Comment by Brdrker** [comment]
Question is which addon. I searched Curse for SpellID and Spell ID to no avail. Any pointers would be helpful. 

And to the prior comment, I looked at the various DB's and I would have to do a singular lookup on each spell, knowing what all of them are beforehand. Also, there are the extra variables in the Ovale spell file like the -15 to cooldown which there is no reference to but I can see these in the existing (corrected) files. It's not just "look it up." It involves a bit more than that.
